edgedb-client/src/builder.rs

```diff
@@ -1138,11 +1140,37 @@ impl Builder {
                         _ => {}
                     }
                 }
+                ServerMessage::ParameterStatusSystemConfig(config) => {
```

Comment:
This is getting long. Let's factor out in a function. Here `server_params` comes handy as `&mut` param.